### PR TITLE
[ru] intermediate mix_tasks - `1.2.2` -> `1.4.1`

### PR DESCRIPTION
**Pull request by IgorPolyakov**

### CODE REVIEW

File: lessons/ru/intermediate/mix_tasks.md [hunk 1]
```diff
@@ -94,7 +94,9 @@ end
 
 ## Загрузка вашего приложения
 
-Mix не запускает наше приложение автоматически или какие-либо его зависимости, что хорошо для многих сценариев использования задач Mix, но что, если нам нужно использовать Ecto и взаимодействовать с базой данных? В этом случае нам нужно убедиться, что приложение, стоящее за Ecto.Repo, запущено. У нас есть два способа справиться с этим: явная инициализация конкретного приложения или инициализация всего нашего приложения, которое, в свою очередь, инициализирует все остальные зависимости.
+Mix не запускает наше приложение автоматически или какие-либо его зависимости, что хорошо для многих сценариев использования задач Mix, но что, если нам нужно использовать Ecto и взаимодействовать с базой данных?
```
Comment (nscyclone) [inline lessons/ru/intermediate/mix_tasks.md:97]:
```suggestion
Mix не запускает наше приложение или какие-либо его зависимости автоматически, что хорошо для многих сценариев использования задач Mix. Но что, если нам нужно использовать `Ecto` и взаимодействовать с базой данных?
```

**Review by nscyclone — CHANGES_REQUESTED** [review]
Thanks @IgorPolyakov 
I'd just like to suggest a minor improvement to enhance readability.

**Comment by IgorPolyakov** [comment]
Thanks for the review, I have made the suggested corrections to enhance readability.

**Review by nscyclone — APPROVED** [review]
Thanks @IgorPolyakov

**Pull request merged**